common-arcana.lic

```diff
@@ -236,4 +236,57 @@ module DRCA
       return parse_mana_message(mana_msg)
     end
   end
+
+  def check_osrel(osrel_no_harness, osrel_amount)
```

Comment:
Yea i think it can be. `check_osrel` could be made into a guard clause in `infuse_om`